drivers/atwinc15x0/atwinc15x0_bsp.c

```diff
@@ -83,7 +83,7 @@ void nm_bsp_register_isr(tpfNmBspIsr pfIsr)
 {
     assert(atwinc15x0);
 
-    DEBUG("%s %p\n", __func__, pfIsr);
+    DEBUG("%s %p\n", __func__, (void*)(uintptr_t)pfIsr);
```

Comment:
```suggestion
    DEBUG("%s %p\n", __func__, (void *)(uintptr_t)pfIsr);
```

I get why here two casts are needed, as casting a function pointer to a pointer is undefined behavior. (They have Harvard architectures in mind, in which the size of a function pointer can be different to the size of a data pointer, as they operate in different address spaces.)